### PR TITLE
docs: Update README

### PR DESCRIPTION
**Pull request by pettax**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,3 +12,23 @@ lint and enforce commit-message formats.
 - commit your additions (follow conventional-commits!)
 - open a pull-request
 - request a review from me + someone else
+
+
+## To install the tools
```
Comment (axelolsson) [inline README.md:17]: Actually they are all present in the `package.json` and the configuration-file already lives in the root so all you need to do is `npm install`.

**Review by axelolsson — CHANGES_REQUESTED** [review]

**Review by axelolsson — COMMENTED** [review]
All commits should follow convention! There is no exception. :)

**Comment by pettax** [comment]
> All commits should follow convention! There is no exception. :)

Even if they are local to a PR (short-lived branch) and about to get squashed?

**Comment by axelolsson** [comment]
> > All commits should follow convention! There is no exception. :)
> 
> Even if they are local to a PR (short-lived branch) and about to get squashed?

Even then yes :)

**Pull request merged**